eox_hooks/actions_handler.py

```diff
@@ -2,24 +2,19 @@
 import logging
 from importlib import import_module
 
+from eox_hooks.actions import *
+
 log = logging.getLogger(__name__)
 
 
 def action_handler(trigger_event, configuration, **kwargs):
     """
     Function that executes actions given a specific configuration.
     """
-    action = action_lookup(
-        module_name=configuration.get("module", "missing_module"),
-        action=configuration.get("action", ""),
-    )
-
-    if not action:
-        return None
-
-    action_result = None
     try:
-        action_result = action(**kwargs)
+        print("AQUIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII")
+        print("-----------> LLAMANDO A LA ACCION")
+        action_result = send_request(**kwargs)
```

Comment:
With this change we are not using action_lookup anymore. Is that ok? or it is just done at this way for testing purposes?

Reply:
You are right, this was only for testing purposes. I should not have uploaded any modification to this file.